package.json

```diff
@@ -67,8 +67,7 @@
       "dist/htmlhint.js"
     ],
     "reporter": [
-      "text",
-      "lcov"
+      "json"
```

Comment:
When I run the coverage locally, I just get the lcov output, is this going to change that, or am I missing something else?

Reply:
This will generate a json file instead of a lcov file but this file is only used for Codecov.